### DISCUSSION TITLE
"sound" field always null when recieving push notifications through FCM on IOS

### DISCUSSION THREAD

**Discussion by ShehabSherbiny** [firebase/flutterfire#12104] [Q&A]
I am currently developing an app, and have run into a weird issue regarding FCM on IOS. For some reason, no matter what i do the "sound" field will always be null, when i recieve the notification on a phone. everything else in the payload will is received normally except "sound".

Looking at the [documentation](https://firebase.google.com/docs/reference/admin/node/firebase-admin.messaging.aps), i am not sure what i am missing. I have tried changing the order of elements in the payload and triple checking my spelling. Has anyone faced a similiar issue? I have included a screenshot of what i recieve in vscode. [vsCode response](https://i.stack.imgur.com/cUxfD.png)

Below is the payload i send from our backend

```
notification: {
  title: inputs.title,
  body: inputs.msg || inputs.message,
  imageURL: undefined
},
android: {
  notification: {
    channel_id: inputs.sound,
    sound: inputs.sound,
    priority: 'high',
    color: inputs.color,
    // NotificationPriority: 'HIGH',
    // clickAction: inputs.priority > 1 ? 'update and open inapp' : undefined,
    // tag: inputs.imageID,
    vibrateTimingsMillis: [1000, 1000]
  },
},
AndroidConfig: {
  sticky: inputs.sticky
},
apns: {
  payload: {
    aps: {
      badge: 50,
      sound: inputs.sound.split('.')[0] + '.aiff',
    }
  }
}

```
Any help would be greatly appreciated!

**Comment by Bryambepz** [comment]
Hi, I try to configure sounds in notifications in a ios device. My questions is if it solved the problem.
I did it once but it was by adding the sound file in xcode and specify the sound name in the configuration of notifications.
